app/models/activity.rb

```diff
@@ -28,7 +28,7 @@ class Activity < ApplicationRecord
   validates :title, :description, presence: true, on: :purpose_step
   validates :sector_category, presence: true, on: :sector_category_step
   validates :sector, presence: true, on: :sector_step
-  validates :status, presence: true, on: :status_step
+  validates :programme_status, presence: true, on: :programme_status_step
```

Comment:
Didn't we decide to leave this as `status_step`..?

Reply:
If I replace line 31 for `validates :programme_status, presence: true, on: :status_step` my validations for the presence of programme status won't work 🤔 When the user creates an activity and does not select any programme status, they don't get the error message that programme status can't be blank. The form just moves onto the next step without complaining...